apps/frontend/src/renderer/components/task-detail/SubtaskActionList.tsx

```
@@ -0,0 +1,489 @@
+}
+
+function SubphaseGroup({ subphase, actions, defaultExpanded = true }: SubphaseGroupProps) {
+  const [isExpanded, setIsExpanded] = useState(defaultExpanded);
+
+  return (
+    <Collapsible open={isExpanded} onOpenChange={setIsExpanded}>
+      <CollapsibleTrigger asChild>
+        <button className={cn(
+          'w-full flex items-center justify-between px-2 py-1.5 rounded-md transition-colors',
+          'hover:bg-secondary/50 text-xs'
+        )}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)